src/main/java/uk/gov/hmcts/reform/em/stitching/batch/DocumentFormatter.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.em.stitching.batch;
+
+import java.io.File;
+import java.io.IOException;
+
+import net.logstash.logback.encoder.org.apache.commons.lang.WordUtils;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.PDPageTree;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+
+import static java.lang.Math.round;
+
+public class DocumentFormatter {
+
+
+    public static File addCoverSheetToDocument(File file) throws IOException {
+
+        String fileName = file.getName();
```

Comment:
Minor point, but as you only use this variable I would not create it and change the method call below to: `addCoversheetTextToFirstPage(document, file.getName()`